CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ x.x.x Release notes (yyyy-MM-dd)
   types.
 * Remove `RLMPropertyTypeArray` in favor of a separate bool `array` property on
   `RLMProperty`/`Property`.
+* Realm Swift APIs that filter or look up the index of an object based on a
```

Comment:
I think you could probably argue that this is a bugfix, since all the things this enables would have been considered incorrect code before. Would you like it retargeted to `master`?